DSharpPlus/Entities/Guild/DiscordGuild.cs

```diff
@@ -121,15 +121,7 @@ public DiscordVoiceRegion VoiceRegion
     /// </summary>
     /// <param name="skipCache">If set to true this method will skip all caches and always perform a rest api call</param>
     /// <returns>Returns null if the guild has no AFK channel</returns>
-    public async Task<DiscordChannel?> GetAfkChannelAsync(bool skipCache = false)
-    {
-        if (this.AfkChannelId is null)
-        {
-            return null;
-        }
-
-        return await GetChannelAsync(this.AfkChannelId.Value);
-    }
+    public async Task<DiscordChannel?> GetAfkChannelAsync(bool skipCache = false) => this.AfkChannelId is null ? null : await GetChannelAsync(this.AfkChannelId.Value, skipCache);

```

Comment:
Again, formatting change that functionally does nothing different. Hinders readability.

Let another reviewer decide whether to keep this or not, I don't know.